javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/context/FieldBackedProvider.java

```diff
@@ -205,7 +197,7 @@ public void visitMethodInsn(
                         .equals(owner)
                     && CONTEXT_GET_METHOD.getName().equals(name)
                     && Type.getMethodDescriptor(CONTEXT_GET_METHOD).equals(descriptor)) {
-                  log.trace("Found context-store access in {}", instrumenterClass.getName());
+                  log.debug("Found context-store access in {}", instrumenterClass.getName());
```

Comment:
we may want to keep this logging at `trace`, but since this was primary culprit of deadlocking earlier, moving it back to debug to verify deadlocking is gone now